<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="side_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(170, 170, 255,.5);
</string>
   </property>
   <widget class="QPushButton" name="patient_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Patients</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/patientdownload .png</normaloff>../Medico_modern/patientdownload .png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dash_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dashboard.png</normaloff>../Medico_modern/dashboard.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="appointment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;

	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}

QPushButton:focus{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>Appointments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/appointmentdownload (1).png</normaloff>../Medico_modern/appointmentdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dentist_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Dentists</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dentistimages.png</normaloff>../Medico_modern/dentistimages.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="payment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Payments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="expense_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Expense</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
font: 75 20pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
font: 22pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string> Medico App </string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="appointment_table">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>1051</width>
     <height>561</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,.85);
font:  12pt &quot;Segoe UI &quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>Date Registered</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Visit time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Visit Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dentist Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Patient ID</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="visit_date">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);

font:  14pt &quot;Segoe UI &quot;;</string>
   </property>
   <property name="text">
    <string>Appointment Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>680</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Images/jonathan-borba-W9YEY6G8LVM-unsplash.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>630</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>640</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>935</x>
     <y>635</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);

font:  14pt &quot;Segoe UI &quot;;
	color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(255, 0, 0);
	background-color: rgb(255, 85, 0);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(255, 0, 0);
	background-color: rgb(255, 85, 0);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Cancel Current Appointment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>640</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Cancel Current Appointment</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>side_bar</zorder>
  <zorder>appointment_table</zorder>
  <zorder>visit_date</zorder>
  <zorder>label_2</zorder>
  <zorder>return_btn</zorder>
  <zorder>label_5</zorder>
  <zorder>cancel_btn</zorder>
 </widget>
 <tabstops>
  <tabstop>appointment_btn</tabstop>
  <tabstop>dentist_btn</tabstop>
  <tabstop>return_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>dash_btn</tabstop>
  <tabstop>patient_btn</tabstop>
  <tabstop>payment_btn</tabstop>
  <tabstop>expense_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
